airbyte-integrations/connectors/source-square/integration_tests/configured_catalog.json

```diff
@@ -148,13 +148,13 @@
       "stream": {
         "name": "orders",
         "json_schema": {},
-        "supported_sync_modes": ["full_refresh"],
+        "supported_sync_modes": ["incremental","full_refresh"],
         "source_defined_cursor": true,
-        "default_cursor_field": ["id"]
+        "default_cursor_field": ["created_at"]
       },
-      "sync_mode": "full_refresh",
-      "cursor_field": ["id"],
-      "destination_sync_mode": "overwrite"
+      "sync_mode": "incremental",
+      "cursor_field": ["created_at"],
```

Comment:
Should the "cursor_field" here match the one at source.py line 393?

Reply:
yes. missed the file while committing changes.